python_modules/dagster-graphql/dagster_graphql/schema/runs.py

```diff
@@ -133,11 +136,23 @@ class Meta:
 class GrapheneRunConfigData(GenericScalar, graphene.Scalar):
     class Meta:
         description = """This type is used when passing in a configuration object
-        for pipeline configuration. This is any-typed in the GraphQL type system,
-        but must conform to the constraints of the dagster config type system"""
+        for pipeline configuration. Can either be passed in as a string (the
+        JSON-serialized configuration object) or as the configuration object itself. In
+        either case, the object must conform to the constraints of the dagster config type system.
+        """
         name = "RunConfigData"
 
 
+def parse_run_config_input(run_config):
+    if isinstance(run_config, str):
+        try:
+            return json.loads(run_config)
+        except JSONDecodeError:
+            return run_config
```

Comment:
hmmm

Reply:
can we at least put this under test to ensure the error is reasonable. If its not might be better to throw here that it was a string and not parseable